config.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+from simple_config import Config
+
+default_settings = {
+    "base_data_dir": '/path/to/base_data_dir',
+    'state_to_analyse': 'state name',
+    'number_of_sample_judgements': 30,
```

Comment:
is there a way to take it from command line instead?

Reply:
Given this would be a growing list, command-line args for a bit more complex config like case types to filter specific to geography would be tricky and hard to maintain. Open for other options though.